src/Utilities.jl

```diff
@@ -1,6 +1,4 @@
-module Utilities
-
-using NLsolve
+import NLsolve: nlsolve, converged
```

Comment:
same in this file, i think we can get rid of the exports maybe_idx, p_v_idx, p_e_idx, @nd_threads, construct_mass_matrix, warn_parallel, checkbounds_p